modules/AOS_PDF_Templates/templateParser.php

```diff
@@ -59,54 +66,51 @@ public function parse_template_bean($string, $key, &$focus)
             if (isset($field_def['name']) && $field_def['name'] != '') {
                 $fieldName = $field_def['name'];
                 if ($field_def['type'] == 'currency') {
-                    $repl_arr[$key . "_" . $fieldName] = currency_format_number($focus->$fieldName, $params = array('currency_symbol' => false));
-                } else {
-                    if (($field_def['type'] == 'radioenum' || $field_def['type'] == 'enum' || $field_def['type'] == 'dynamicenum') && isset($field_def['options'])) {
-                        $repl_arr[$key . "_" . $fieldName] = translate($field_def['options'], $focus->module_dir, $focus->$fieldName);
+                    $repl_arr[$key . "_" . $fieldName] = currency_format_number($focus->$fieldName,
```

Comment:
Please fix code formatting:

If using multiple line functions - use the following format:
```
    $repl_arr[$key . "_" . $fieldName] = currency_format_number(
         $focus->$fieldName,
         $params = array(
             'currency_symbol' => false
         )
    );
```